onedal/linear_model/linear_model.py

```diff
@@ -21,15 +21,18 @@
 
 from daal4py.sklearn._utils import get_dtype, make2d
 
-from ..common._base import BaseEstimator as onedal_BaseEstimator
```

Comment:
Better to leave onedal_BaseEstimator naming

Reply:
what is the reason of this renaming? does it make sense to rename class itself if we plan to rename it in every import?